### ISSUE TITLE
[FEATURE]: Translate image Caption and Alt Text fields

### ISSUE THREAD

**Issue by jsadoski-rockhall** [Fekide/strapi-plugin-translate#405]
**Is your feature request related to a problem? Please describe.**
When creating a Media field, from the "Advanced Settings" tab, under the "Configure automated translation for this field?" dropdown, we can select "Translate". However, this doesn't seem to text fields related to Media content, namely Caption and Alt Text.

**Describe the solution you'd like**
Translate the fields for Media assets, since Alt Text and Captions should be displayed with the rest of the content in whatever front-end you use.

**Describe alternatives you've considered**
We could just copy the assets ourselves and translate them, but that would be a very manual process.

**Additional context**
I would be fine with the asset being duplicated with translated fields, if that's the best way to do it.
